config/default.yml

```
@@ -3623,6 +3623,11 @@ Style/LambdaCall:
     - call
     - braces
 
+Style/LegacyHash:
+  Description: 'Avoid Hash[] in favor of ary.to_h or literal hashes.'
+  Enabled: pending
+  VersionAdded: '1.10'
```

```suggestion
  VersionAdded: <<next>>
```

We don't hardcode next versions anymore, `<<next>>` is metadata that is automatically updated when a release is made.